src/components/ItemDetail.jsx

```diff
@@ -0,0 +1,141 @@
+import { useEffect, useState, useCallback } from "react";
+import { getItemDetail, getComments } from "../api";
+import { useParams } from "react-router-dom";
+import styles from "./ItemDetail.module.css";
+
+const itinialItem = {
+  name: "",
+  description: "",
+  price: "",
+  tags: [],
+  images: [],
+  favoriteCount: 0,
+};
+
+function ItemDetail() {
+  const { productId } = useParams();
+  const [item, setItem] = useState(itinialItem);
```

Comment:
```suggestion
  const [item, setItem] = useState(initialItem);
```

vscode를 사용하신다면 [codeSpellChecker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)를 사용하시면 오타를 잘 잡아낼 수 있답니다!